packages/react-instantsearch-hooks/scripts/transforms/rish-to-ris.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-shadow */
+
+export default function transform(file, api, options) {
+  const j = api.jscodeshift;
+  const printOptions = options.printOptions || { quote: 'single' };
+  const root = j(file.source);
+
+  const replaceImports = (from, to) => (root) =>
+    root
+      .find(j.ImportDeclaration)
+      .filter(
+        (path) =>
+          path.node.source.value === from ||
+          path.node.source.value.startsWith(`${from}/`)
+      )
+      .forEach((sourceImport) => {
+        j(sourceImport).replaceWith(
+          j.importDeclaration(
+            sourceImport.node.specifiers,
+            j.stringLiteral(sourceImport.value.source.value.replace(from, to))
+          )
+        );
+      });
+
+  const replaceHooks = replaceImports(
+    'react-instantsearch-hooks',
+    'react-instantsearch-core'
+  );
+  const replaceHooksWeb = replaceImports(
+    'react-instantsearch-hooks-web',
+    'react-instantsearch'
+  );
+
+  replaceHooks(root);
+  replaceHooksWeb(root);
+
```

Comment:
Log to remind users of changing packages in their `package.json`

Reply:
Can't do this but we can add an `echo Remember to update package.json` in README if they just copy/paste

Reply:
why can't we do it? do logs get suppressed? We can use a similar "send warning only once" system maybe? (or is the file executed totally separate)?